public/apps/configuration/panels/audit-logging/audit-logging-edit-settings.tsx

```diff
@@ -229,9 +229,17 @@ export function AuditLoggingEditSettings(props: AuditLoggingEditSettingProps) {
     );
   };
 
+  let content;
+
   if (props.setting === 'general') {
-    return renderGeneralSettings();
+    content = renderGeneralSettings();
+  } else {
+    content = renderComplianceSetting();
   }
 
-  return renderComplianceSetting();
+  return (
+    <>
```

Comment:
Why need a fragment here?

Reply:
Not needed. Will address in follow up PRs.